src/Forem.Api/Models/Article.cs

```diff
@@ -16,38 +20,66 @@ public class Article
 		public string Description { get; set; }
 		[JsonProperty("cover_image")]
 		public Uri CoverImage { get; set; }
+
+		[JsonProperty("readable_publish_date")]
+		public string ReadablePublishDate { get; set; }
+
 		[JsonProperty("social_image")]
 		public Uri SocialImage { get; set; }
-		[JsonProperty("created_at")]
-		public DateTime CreatedAt { get; set; }
-		[JsonProperty("published_at")]
-		public DateTime PublishedAt { get; set; }
-		[JsonProperty("edited_at")]
-		public DateTime? EditedAt { get; set; }
+		
 		[JsonProperty("tag_list")]
+		public string TagList { get; set; }
+
+		[JsonProperty("tags")]
 		[JsonConverter(typeof(MultiTypeTagConverter))]
-		public IEnumerable<string> TagList { get; set; }
+		public IEnumerable<string> Tags { get; set; }
```

Comment:
The problem with changing this is that while `tags` (comma-separated string) and `tag_list` (array) is available on these APIs:
- https://developers.forem.com/api/v0#tag/articles/operation/getArticles
- https://developers.forem.com/api/v0#tag/articles/operation/getLatestArticles

It is the opposite way around on this API (`tags` as array, `tag_list` as comma-separated string):
- https://developers.forem.com/api/v0#tag/articles/operation/createArticle
- https://developers.forem.com/api/v0#tag/articles/operation/getArticleById
- https://developers.forem.com/api/v0#tag/articles/operation/updateArticle
- https://developers.forem.com/api/v0#tag/articles/operation/getArticleByPath

And only `tag_list` is available on these APIs (as an array):
- https://developers.forem.com/api/v0#tag/articles/operation/getUserArticles
- https://developers.forem.com/api/v0#tag/articles/operation/getUserPublishedArticles
- https://developers.forem.com/api/v0#tag/articles/operation/getUserUnpublishedArticles
- https://developers.forem.com/api/v0#tag/articles/operation/getUserAllArticles

Because of that last part (some APIs only having `tag_list` and those APIs having it in array-form) is that by changing it to `tags` is that this won't be set but also the string property `TagList` (via `tag_list`) won't be able to deserialize it. I think leave this as `tag_list` and the string one as `tags`.

Reply:
Like this, yeas?

![image](https://user-images.githubusercontent.com/43586181/209906644-14694357-89a6-46e3-bef9-8fcd2f1013a2.png)

Reply:
Yeah, that. It sucks that we need to do this but until they fix up the API, we're stuck.